code/game/objects/items/weapons/autopsy.dm

```diff
@@ -79,8 +79,8 @@
 	set category = "Object"
 	set src in view(usr, 1)
 	set name = "Print Data"
-	if(usr.stat || !(ishuman(usr)))
-		usr << "No."
+	if(usr.stat)
+		usr << "You must be conscious to do that!"
```

Comment:
That check was there because of mice what spam with printing Reports.
Should be replaced with something like `mob.IsAdvancedToolUser()`